scripts/Text Grammar Corrector/grammer bot.py

```diff
@@ -0,0 +1,47 @@
+# Imports
+import language_tool_python
+
+# Initial API
+tool = language_tool_python.LanguageToolPublicAPI("en-US")
+
+# initalizae filename
+FILENAME = "input.txt"
```

Comment:
I think it is better that we ask the user for input either during runtime or with command line arguments using `argparse`.

Reply:
Done